### PR TITLE
Delete a lot of legacy code

### PR DESCRIPTION
**Pull request by jcmorrow**
There's a lot of strange history here. The basic approach was:

- delete the `src/devtools/client/debugger/dist/` directory, except for a few still used packages
- transfer those packages over to `src/devtools/packages` and, in some cases, either delete them or fold them into `src/devtools/shared` if they are really just a single source file
- re-create a couple of files from `vendor.js`, but delete the rest
- delete copies of the mozilla license that were in subdirectories (I assume the top-level license covers everything)
- delete READMEs when they seemed unhelpful
- delete old `.rc` files

### CODE REVIEW

File: src/devtools/client/inspector/inspector.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 // "use strict";
 
-// const Services = require("Services");
+// const Services = require("devtools/shared/services");
 const EventEmitter = require("devtools/shared/event-emitter");
 // const InspectorStyleChangeTracker = require("devtools/client/inspector/shared/style-change-tracker");
 // const { log } = require("protocol/socket");
```
Comment (jasonLaster) [inline src/devtools/client/inspector/inspector.ts:10]: lol so much red in the inspector dir

File: src/devtools/client/shared/inplace-editor.js [hunk 2]
```diff
@@ -666,7 +666,8 @@ InplaceEditor.prototype = {
    */
   _parseCSSValue: function (value, offset) {
     /* eslint-disable max-len */
-    const reSplitCSS = /(url\("?[^"\)]+"?\)?)|(rgba?\([^)]*\)?)|(hsla?\([^)]*\)?)|(#[\dA-Fa-f]+)|(-?\d*\.?\d+(%|[a-z]{1,4})?)|"([^"]*)"?|'([^']*)'?|([^,\s\/!\(\)]+)|(!(.*)?)/;
+    const reSplitCSS =
```
Comment (jasonLaster) [inline src/devtools/client/shared/inplace-editor.js:669]: wonder what the inplace editor is 😉

File: src/ui/components/App.css [hunk 3]
```diff
@@ -107,7 +107,7 @@
 @import url("devtools/client/themes/components-frame.css");
 @import url("devtools/client/themes/markup.css");
 @import url("devtools/client/themes/common.css");
-@import url("devtools/client/debugger/dist/vendors.css");
+/* @import url("devtools/client/debugger/dist/vendors.css"); */
```
Comment (jasonLaster) [inline src/ui/components/App.css:110]: lets burn this line

File: tsconfig.json [hunk 4]
```diff
@@ -43,7 +43,7 @@
       "*": [
         "src/*",
         "src/devtools/client/debugger/dist/*",
-        "src/devtools/client/debugger/packages/*",
+        "src/devtools/packages/*",
```
Comment (jasonLaster) [inline tsconfig.json:46]: can we delete these two line?

```
        "src/devtools/client/debugger/dist/*",
        "src/devtools/packages/*",
```

**Review by jasonLaster — APPROVED** [review]
👍 with passing tests

File: tsconfig.json (hunk 4 above)
Comment (jcmorrow) [reply tsconfig.json]: done!

**Comment by jcmorrow** [comment]
I need to figure out why the mock tests are all failing 🤔 they pass locally

**Comment by jcmorrow** [comment]
> I need to figure out why the mock tests are all failing 🤔 they pass locally

Turns out, they are borked on `master` too 😢 . Ignoring for now, I'll get to that!

**Pull request merged**